boards/riscv/mpfs_icicle/support/mpfs250t.resc

```diff
@@ -0,0 +1,17 @@
+:name: Mi-V
+:description: This script is prepared to run Zephyr on a Mi-V RISC-V board.
+
+$name?="Mi-V"
```

Comment:
`Mi-V` could also refer to the `m2gl025_miv` board. Maybe `mpfs_icicle` would be a better suit here?